proto/streamlit/proto/Button.proto

```diff
@@ -19,5 +19,5 @@ syntax = "proto3";
 message Button {
   string id = 1;
   string label = 2;
-  bool default = 3;
+  bool value = 3;
```

Comment:
Per @tvst our practice is to make changes to protobufs by creating new entries and "retiring" the old ones.

So, you would make a new numerical entry for value, and then at the bottom of the protobuf declaration you put:

reserved 3;
reserved default;